src/screens/AboutApp/AboutApp.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { ScrollView, View, Text, Image } from "react-native";
+import { translate } from "../../i18n";
+import { Button } from "../../components";
+
+import styles from "./styles";
+
+const logo = require("../../assets/logotype/Logo.png");
+const epamLogo = require("../../assets/logotype/epam-logo.png");
+
+interface AboutAppProps {
+  version: string,
+  phone: string,
+  email: string,
+}
+
+const AboutApp: React.FC<AboutAppProps> = ({ version, phone, email }) => (
+  <ScrollView showsVerticalScrollIndicator={false} style={styles.container}>
+    <View style={styles.margin}>
+      <View style={styles.logo}>
+        <Image resizeMode="contain" style={styles.image} source={logo}/>
+        <Text style={styles.version}>
+          {`${translate("aboutApp.version")} ${version}`}
+        </Text>
+      </View>
+      <Text style={styles.text}>{translate("aboutApp.description")}</Text>
+      <Button
+        caption={translate("aboutApp.readMore")}
+        appearance="Light"
+        wrapperStyles={styles.buttonWrapper}
+        onPress={() => {
+          // TODO add logic when we will know what exactly should do current button
+        }}
+      />
+      <View style={styles.article}>
+        <Text style={styles.title}>{translate("aboutApp.aboutCenter")}</Text>
```

Comment:
Font in the app is not the same as in invision. And I don't see Lato font in assets

<img width="1148" alt="Screen Shot 2020-01-08 at 10 36 05 PM" src="https://user-images.githubusercontent.com/58725560/72009787-a507e600-3267-11ea-87f2-ffb9f34133cc.png">

Reply:
Fonts will be add in another PR